pilot/cmd/pilot-agent/main.go

```diff
@@ -287,6 +281,31 @@ var (
 	}
 )
 
+//ToDo: add missing tests
```

Comment:
Please add comments for what it does.

And when leave a TODO, please add the corresponding person for this TODO.